### PR TITLE
Correction in "Develop ASP.NET Core MVC apps"

### PR DESCRIPTION
**Pull request by widjesh**
## Summary

I changed this 

"
app.UseMvc(routes =>;
{
    routes.MapRoute("default","{controller=Home}/{action=Index}/{id?}");
});
"

 into this

"
app.UseMvc(routes =>
{
    routes.MapRoute("default","{controller=Home}/{action=Index}/{id?}");
});
"

### CODE REVIEW

File: docs/architecture/modern-web-apps-azure/develop-asp-net-core-mvc-apps.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ At its heart, ASP.NET Core apps map incoming requests to outgoing responses. At
 ASP.NET Core MVC apps can use conventional routes, attribute routes, or both. Conventional routes are defined in code, specifying routing _conventions_ using syntax like in the example below:
 
 ```csharp
-app.UseMvc(routes =>;
```
Comment (widjesh) [inline docs/architecture/modern-web-apps-azure/develop-asp-net-core-mvc-apps.md]: the ";" should not be there

**Review by widjesh — COMMENTED** [review]
the ";" should not be there

**Review by Youssef1313 — CHANGES_REQUESTED** [review]
Shouldn't the same fix be applied to this line ?
`_connection.On("receiveMessage", (arguments) =>;`

**Review by Youssef1313 — CHANGES_REQUESTED** [review]
I also think that there shouldn't be a `\`s in this line:
`Console.WriteLine(\$"{arguments\[0\]} said: {arguments\[1\]}");`

**Update:**

I've made a separate [PR](https://github.com/dotnet/docs/pull/13819) for this `\` issue.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thank you, @widjesh, for contributing to the dotnet/docs repo and correcting the syntactically incorrect line of code. There are a number of other compiler errors in the code. Could you correct them also? They are:
- line 205: should begin with `public class` to define the type.
- line 471: no `;` following `=>`

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**